tests/test_js_signing.py

```diff
@@ -0,0 +1,55 @@
+"""
+Various tests related to validation for automated signing.
+"""
+
+from js_helper import TestCase
```

Comment:
Yeah, if we have opportunities to improve the code we're copying/modifying anyways... let's take them ;)